tests/metrics/classification/test_f_beta.py

```diff
@@ -1,81 +1,76 @@
-import os
-import pytest
-import torch
-import numpy as np
-from collections import namedtuple
-
 from functools import partial
 
-from pytorch_lightning.metrics import Fbeta
+import numpy as np
+import pytest
+import torch
 from sklearn.metrics import fbeta_score
 
-from tests.metrics.utils import compute_batch, setup_ddp
-from tests.metrics.utils import NUM_BATCHES, NUM_PROCESSES, BATCH_SIZE, NUM_CLASSES, THRESHOLD
-
+from pytorch_lightning.metrics import Fbeta
```

Comment:
can we have this as `FBeta` to be consistent?

Reply:
@ananyahjha93

Reply:
if you agree I can integrate it in #4152